pkg/cmd/samples/create.go

```diff
@@ -65,7 +66,12 @@ func (cc *CreateCmd) runCreateCmd(cmd *cobra.Command, args []string) error {
 		Git:    gitpkg.Operations{},
 	}
```

Comment:
maybe out of scope of this diff, but to me this is an implementation detail that should live inside the package, not called here.

Reply:
Do you mean the construction of the Samples object? I can change it to follow this pattern using `New` : https://github.com/stripe/stripe-cli/blob/3fbf545adac182d1374bbe04e5c84a22f0a04851/pkg/logtailing/tailer.go#L94-L107

Reply:
No, more about the fact that you have to initialize a `sample` object, only with default, to then call functions that feel like they could be package function instead of instance functions.

Basically instead of doing
```
sample := samples.Sample{}
sample.CreateSample()
```

it seems like the interface would be cleaner with
```
samples.CreateSample()
```
directly.

Reply:
This is not blocking by the way, mostly just nit/personal perf.